ephios/plugins/questionnaires/urls.py

```diff
@@ -0,0 +1,72 @@
+from django.urls import path
+
+from ephios.plugins.questionnaires.views import (
+    AggregateAnswerView,
+    QuestionArchiveView,
+    QuestionCreateView,
+    QuestionDeleteView,
+    QuestionListView,
+    QuestionUpdateView,
+    SavedAnswerDeleteView,
+    SavedAnswerListView,
+    SavedAnswerUpdateView,
+)
+
+app_name = "questionnaires"
+urlpatterns = [
+    path(
+        "questions/",
+        QuestionListView.as_view(),
+        name="question_list",
+    ),
+    path(
+        "questions/archived/",
+        QuestionListView.as_view(show_archived=True),
+        name="question_list_archived",
+    ),
+    path(
+        "questions/add/",
```

Comment:
damn I never noticed. Sometimes we add, sometimes we create. I vote for "create" everywhere.